pylib/cqlshlib/cql3handling.py

```diff
@@ -380,6 +380,11 @@ def dequote_value(cqlword):
                             ( ender="," [propmapkey]=<term> ":" [propmapval]=<term> )*
                       ender="}"
                     ;
+<propertyOrOption> ::= <property>
+                     | "ALL"
+                     | "INDEX"
```

Comment:
Reading the CEP, I think these names should be plural right? `INDEXES` and `TRIGGERS`?